### PR TITLE
fix compile warning originating in cpp skeleton

### PR DESCRIPTION
**Pull request by GitMensch**
`gcount()` is of type `streamsize` which has a system-dependent size with at least `int`. Because of the `int max_size` limit no bigger value will be returned, the explicit cast fixes warnings from compilers/static analyzers.

As an alternative `LexerInput` may be changed to have a return and parameter of type `streamsize`, but this would need more changes and likely create more warnings in other places.

This issue originates from https://github.com/lexxmark/winflexbison/issues/73 where more details can be found (this is not a MSVC only issue).

### CODE REVIEW

**Review by Mightyjo — APPROVED** [review]
Looks good. Your reasoning makes sense. I double-check that streamsize is defined as a "signed, integral type", so with our limit to int max_size this seems right. 

'git clean -xf' ... 'make distcheck' from a clean maintainer tree build cleanly.

**Comment by GitMensch** [comment]
@Mightyjo Thanks for the review, so it is up to @westes to merge.

Side question @westes: maybe there's someone that could be added as a developer to this repo and could work on handling the PRs (currently 39 open) / issues (>100)?

**Pull request merged**
